images/base/r-devel.md

```diff
@@ -42,7 +42,7 @@ These prerelease version R can be executed with `RD`(for R-devel) or `RP`(for R-
 - `rocker/drp` has R-patched installed.
 - `rocker/r-devel-san` and `rocker/r-devel-ubsan-clang` provide R development binaries with Sanitizer support,
   for memory checking during R package development[^r-exts].
-  `rocker/r-devel-san` uses clang, and `rocker/r-devel-ubsan-clang` uses gcc.
+  `rocker/r-devel-san` uses gcc, and `rocker/r-devel-ubsan-clang` uses clang.
```

Comment:
The bigger problem is that one of the two hasn't built in some time.  I revisited it a few months (?) ago and it still failed to build. 

(The other real problem is that it would help if CRAN published a Dockerfile but alas we're not there yet...)

Reply:
completely agree, what's more... images that CRAN could be publishing would finally resolve various problems of not being able to reproduce CRAN issues locally.